### ISSUE TITLE
Search plugin?

### ISSUE THREAD

**Issue by theerror** [bckp/synology-webshare#9]
Hi, would it be possible to implement also search plugin, so it would be possible to use Download station also for searching on WS?

**Comment by theerror** [comment]
ahhh, sorry, it's there already... .

https://github.com/vencam/synology-download-station-webshare.cz.dlm

EDIT: but, looks not working :/

**Comment by bckp** [comment]
Yeah, i saw it and I know it is not working, i can for sure create search into my plugin (it can be harder to support Synology RTM) but so far i do not see point, as working with webshare show you previews and other informations, i cant put into plugin... But yeah, i will keep this open and if have little time, i will implement it

**Comment by walkercz** [comment]
Pokud to nějak přispěje, rád pomůžu s testováním SRM 1.3

**Comment by bckp** [comment]
@walkercz thanks, no need so far, as i already have some synology routers, but not AX6600 :( can you test in future on that, or you have lower model?

**Comment by walkercz** [comment]
Yes I have, let me know if you need anything.
